constants.yml

```diff
@@ -114,6 +118,14 @@ company_status_detail:
     'petition-to-restore-dissolved' : "Petition to restore dissolved"
     'transformed-to-se' : "Transformed to SE"
     'converted-to-plc' : "Converted to PLC"
+overseas_acct_type:
```

Comment:
could merge overseas_account_type and overseas_account_status together to have something like:
overseas_accounts:
    type:
        'type1' : "some text"
    status:
        'status1' : "some status text"